sqlglot/parser.py

```diff
@@ -1181,7 +1181,7 @@ def extend_props(temp_props: t.Optional[exp.Expression]) -> None:
             table_parts = self._parse_table_parts(schema=True)
 
             # exp.Properties.Location.POST_NAME
-            if self._match(TokenType.COMMA):
+            if self._match(TokenType.COMMA) or self._match(TokenType.ON, advance=False):
```

Comment:
don't add this, just remove the if

Reply:
I think it will always raise in the property parser then:
```
        if self._match_texts(self.PROPERTY_PARSERS):
            parser = self.PROPERTY_PARSERS[self._prev.text.upper()]
            try:
                return parser(self, **{k: v for k, v in kwargs.items() if v})
            except TypeError:
                self.raise_error(f"Cannot parse property '{self._prev.text}'")
```

Reply:
just do 

self._match(TokenType.COMMA)

i removed this in my local and it worked fine